src/clock.rs

```diff
@@ -6,11 +6,15 @@ use e310x_hal::{
     time::Hertz,
 };
 
-#[cfg(any(feature = "board-hifive1", feature = "board-hifive1-revb"))]
+#[cfg(any(
+    feature = "board-hifive1",
+    feature = "board-hifive1-revb",
+    feature = "board-lofive-r1",
+))]
 /// Configures clock generation system.
 ///
-/// For HiFive1 and HiFive1 Rev B boards external oscillators are enabled for
-/// both high-frequency and low-frequency clocks.
+/// For HiFive1, HiFive1 Rev B, and LoFive R1 boards, the external oscillators
+/// are enabled for both high-frequency and low-frequency clocks.
```

Comment:
I believe that LoFive R1 does not have any external oscillator for the low-frequency clock:
https://raw.githubusercontent.com/wiki/mwelling/lofive/lofive-r1-schematic.png

Reply:
Oh, so should I have added the R1 to the docs on the other function then? Honestly, I just assumed that it'd be grouped with the HiFive1 Rev B because they both use the G002...